airflow/dag_processing/manager.py

```diff
@@ -366,6 +366,10 @@ class DagFileProcessorManager(LoggingMixin):
     :param async_mode: whether to start the manager in async mode
     """
 
+    DEFAULT_FILE_STAT = DagFileStat(
```

Comment:
Minor tweak: this is a named tuple, this immutable. So I'm reusing the same static instance over and over, rather than periodically creating new ones.